### PR TITLE
chore(STONEO11Y-61): add makefile

### PR DESCRIPTION
**Pull request by yftacherzog**
Run tests using makefile. This will allow developers running test prep steps together with the tests themselves using `make all`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.PHONY: all
+all: prepare lint test_rules
```
Comment (eisraeli) [inline Makefile:2]: Don't you want to run `pint` as well ?

Comment (yftacherzog) [reply Makefile:2]: I ran it locally and noticed it reinstall pint every time. It also not on gitignore so having it here increases the risk of it being pushed to git. I opened a ticket to address those. Once addressed, we should add it as well here.

BTW, target `all` is only run in local dev envs. on CI, the targets are run independently according to the ci step, so pint is running there.

**Review by avi-biton — APPROVED** [review]

**Comment by avi-biton** [comment]
LGTM

**Review by eisraeli — APPROVED** [review]
LGTM

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**